.github/workflows/pr.yaml

```diff
@@ -1,20 +1,133 @@
-name: Lint PR
+name: Pull Request
 
 on:
   pull_request_target:
```

Comment:
It does, but we can make it so that labeling is skipped for drafts. Good suggestion. I want to keep the title lint for drafts just so that there's no race between someone hitting "ready for review" followed by "merge" before the action has run (maybe it's not possible but if it is.. 😄)

Reply:
Fixed in 243555d.